sandbox-backend/src/main/java/com/exadel/sandbox/team5/configs/security/util/JwtUtil.java

```diff
@@ -0,0 +1,58 @@
+package com.exadel.sandbox.team5.configs.security.util;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtUtil {
+
+    //    @Value("${app.jwtSecret}")
+    private String jwtSecret = "salt";
+    //    @Value("${app.jwtExpirationMs}")
+    private long jwtExpirationMs = 100000;
```

Comment:
You can use @value with default value, for example `@Value("${app.jwtSecret:100000}`